pkgs/top-level/release.nix

```diff
@@ -24,6 +25,8 @@ with import ./release-lib.nix { inherit supportedSystems scrubJobs nixpkgsArgs;
 let
 
   systemsWithAnySupport = supportedSystems ++ limitedSupportedSystems;
+  platformsWithAnySupport = map lib.systems.elaborate
```

Comment:
Maybe name this relating to bootstrap tools since it's the only place this is used.